src/status_im/contexts/settings/wallet/network_settings/testnet_mode/view.cljs

```diff
@@ -0,0 +1,54 @@
+(ns status-im.contexts.settings.wallet.network-settings.testnet-mode.view
+  (:require [quo.core :as quo]
+            [status-im.contexts.settings.wallet.network-settings.testnet-mode.style :as style]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn on-cancel-change
+  []
+  (rf/dispatch [:hide-bottom-sheet]))
+
+(defn logout
+  []
+  (rf/dispatch [:logout]))
+
+(defn on-confirm-change
+  [enable?]
+  (rf/dispatch [:hide-bottom-sheet])
+  (rf/dispatch [:profile.settings/profile-update :test-networks-enabled? enable? {:on-success logout}]))
```

Comment:
I'd change the `profile-update` event to accept a single map instead of multiple args.

Reply:
I agree that this event should probably be refactored 🛠️ 
Though I would prefer to avoid doing that in this PR, perhaps we can create a separate issue?
I mostly want a separate issue so that we can use `skip-manual-qa` for this PR, and potentially the `request-manual-qa` when/if we refactor the `:profile.settings/profile-update` event.